app/lib/services/ecf/application_updater.rb

```diff
@@ -0,0 +1,18 @@
+module Services
+  module Ecf
+    class ApplicationUpdater
+      def initialize(data)
+        @data = data
+      end
+
+      def call
+        id = @data["attributes"]["id"]
+        lead_provider_approval_status = @data["attributes"]["lead_provider_approval_status"]
+        participant_outcome_state = @data["attributes"]["participant_outcome_state"]
+
+        application = Application.find_by(ecf_id: id)
```

Comment:
Here, I would rather prefer to have `ActiveRecord::RecordNotFound` exception as it should never happen. If happens, there is something wrong with the data and we will investigate.